MTM1M3Test.py

```diff
@@ -212,13 +204,10 @@ async def _raising(self):
                     raising_rate = (
                         abs(last_raising_ims.zPosition - ims.zPosition) * M2UM
                     ) / mdur
-                    self.max_raising_rate = max(
-                        self.max_raising_rate, raising_rate
-                    )
+                    self.max_raising_rate = max(self.max_raising_rate, raising_rate)
                 last_raising_ims = ims
                 if not (
-                    self.m1m3.evt_detailedState.get().detailedState
-                    == raisingState
+                    self.m1m3.evt_detailedState.get().detailedState == raisingState
```

Comment:
I believe we want to use `.aget()` instead of `.get()` to avoid race conditions.

Reply:
Which race conditions? I need the current detailed state, I don't want the code to wait if one isn't available. There isn't any race associated with get, it simply retrieve value stored - no other code can write to variable location, that's how async works.